nix/packages/cardanow-ts.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, config
+, dream2nix
+, ...
+}: {
+  imports = [
+    dream2nix.modules.dream2nix.nodejs-package-lock-v3
+    dream2nix.modules.dream2nix.nodejs-granular-v3
+  ];
+  deps = { nixpkgs, ... }: {
+    inherit
+      (nixpkgs)
+      gnugrep
+      stdenv
+      ;
+  };
+  nodejs-package-lock-v3 = {
+    packageLockFile = ../../package-lock.json;
+  };
+  nodejs-granular-v3 = {
+    buildScript = ''
+      tsc ./main.ts
+      mv main.js main.js.tmp
+      echo "#!${config.deps.nodejs}/bin/node" > main.js
+      cat main.js.tmp >> main.js
+      chmod +x ./main.js
+      patchShebangs .
+    '';
+  };
+
+  name = lib.mkForce "cardanow-ts";
+  version = lib.mkForce "0.0.0.1";
```

Comment:
I wonder if these options aren't automatically set by dream2nix nodejs modules if you add them to the `package.json` file.

Reply:
If they are not you an use `lib.importJSON` to read that file (after having added them there) and set the options
Do what you prefer, it was mainly for having a single source of truth for the version.

Reply:
At the moment they are not duplicates because there is no name nor version in the `package.json`. I meant that it could make sense put them there instead, since probably dream2nix will get them automatically from there.